halo/_utils.py

```diff
@@ -115,6 +115,25 @@ def decode_utf_8_text(text):
         return text
 
 
+def encode_utf_8_text(text):
+    """Decode the text from utf-8 format
+
+     Parameters
```

Comment:
Remove extra spacing here:

```suggestion
    Parameters
```